packages/prisma/migrations/20220308100145_/migration.sql

```diff
@@ -0,0 +1,44 @@
+-- AlterTable
+ALTER TABLE "EventType" ADD COLUMN     "successRedirect" TEXT;
+
+-- RenameIndex
+ALTER INDEX "Booking.uid_unique" RENAME TO "Booking_uid_key";
+
+-- RenameIndex
+ALTER INDEX "DailyEventReference_bookingId_unique" RENAME TO "DailyEventReference_bookingId_key";
+
+-- RenameIndex
+ALTER INDEX "DestinationCalendar.bookingId_unique" RENAME TO "DestinationCalendar_bookingId_key";
```

Comment:
For some reason, Prisma keeps generating these ALTER INDEX lines, it happened on my last PR